Library/Homebrew/os/mac/include/llvm/__config_site

```diff
@@ -0,0 +1,7 @@
+#ifndef _HOMEBREW_LIBCPP___CONFIG_SITE
```

Comment:
Feels like this file should probably be under `Library/Homebrew/extend/ENV/` somewhere

Reply:
It's a thing injected by the superenv so probably should be in `shims/super` somewhere

Reply:
Could just do `shims/mac/super/include` for now.

I am hoping this is a short term workaround while we try to figure out a fix in LLVM formula.

If it looks like we will need to carry for longer term or we need to shim in other headers (e.g. maybe for expat), then I can update directory structure.